tests/cpp/test_matmul.cpp

```diff
@@ -3648,7 +3648,13 @@ INSTANTIATE_TEST_SUITE_P(
     testing::ValuesIn(kAllSupportedMmaLayout),
     mmaLayoutName);
 
-using HopperMatmulTest = HopperBase;
+class HopperMatmulTest : public HopperBase {
```

Comment:
Enabled TensorIndexer for this fixture.

The only difference I found is use of `threadIdx.y` when `blockDim.y` is 1, which is automatically replaced with 0 in TensorIndexer. Actual diff can be seen [here](https://www.diffchecker.com/cjnk2WCH/)